.github/workflows/lint.yml

```diff
@@ -27,6 +27,9 @@ jobs:
           github-token: ${{ secrets.github_token }}
         if: env.GIT_DIFF
 
+      - name: Run make lint-fix for multiplexer
+        run: make lint-fix -C multiplexer
+
```

Comment:
[question] why doesn't the above step

```
- uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd #v7.0.0
        with:
          version: v2.0.1
          args: --timeout 10m
          github-token: ${{ secrets.github_token }}
        if: env.GIT_DIFF
```

result in an error if there are lint issues in the multiplexer code?